### ISSUE TITLE
A lot of Kerning inconsistencies

### ISSUE THREAD

**Issue by oomek** [jpt/barlow#59]
First of all I love this font, but it breaks my heart when I see that kerning tables are still half baked. At least it's what's FontLab is showing. Freetype2 library is making it even worse at small sizes, unfortunately.

![image](https://user-images.githubusercontent.com/2974860/72213608-8c0e6780-34e9-11ea-920c-3ff9127bfa16.png)

**Comment by forsgren** [comment]
+1

Was about to use this, but "2019" also ended up with poor kerning. Hoping this can be fixed.

**Comment by jpt** [comment]
Hi all, thanks for pointing this out. I think this broke because I brought in kerning from iKern but didn't bring in the classes? I will have to look. But honestly it's not a lot of masters, I should just re-kern the whole thing once I fix the other glyph-level issues.

**Comment by oomek** [comment]
Thanks for reply. That's what I did in FontLab ;) Manually re-kerned the missing pairs

**Comment by jpt** [comment]
this is fixed in https://github.com/jpt/barlow/tree/1.5 - I have rekerned the font. new font binaries coming soon.

**Comment by jpt** [comment]
binaries available here for testing: https://github.com/jpt/barlow/tree/1.5/fonts

**Comment by forsgren** [comment]
Seems to be improved now 👍

**Comment by forsgren** [comment]
There still seems to be some issues in the black italic variant. A few of the pairs I think it is a bit off between:

- "SW"
- "OY"
- "SIG" (not quite sure about this one, to me it's the capital i that looks a bit "off center" between S and G)
- "ON"
- "CO"

**Comment by jpt** [comment]
Thanks for reporting - along with #74 I think there is a growing case to seriously look into #11. As it is, the kerning is probably not even receiving the correct offset as the slant is applied in the export process.

**Comment by forsgren** [comment]
I had a quick look among the upright characters as well. Based on the previous listing I find that at least these are quite noticeable there as well:

- "SW"
- "OY"

These are less so:

- "ON"
- "SIG"
- "CO"

The slant being applied in the export can be one potential cause, but I would guess there still might be some issues among the upright versions.

This is in no way a comprehensive "inspection", just a few I noticed in a current personal design I'm working on :)

**Comment by jpt** [comment]
yeah, there are multiple issues with the way I set up Kern On (as based on previously set pairs) — this is all on the list :)
